media/js/firefox/australis/tour.js

```diff
@@ -3,6 +3,12 @@
 
     var firstTime = 'True';
 
+    // GA tracking for Firstrun tests
```

Comment:
Hm, good calls. I believe verifying `?f=34` is sufficient, as only downloaders of 35.0.1 will get the funnelcake param appended.